ONSdigital/eq-author-app
# Add validation for unselected answer

### What is the context of this PR?

Adds a red box and error message to routing and skip condition selectors

Matching the design found in corresponding ticket

Branch name is incorrect, accidentally looked at the wrong ticket

### How to review 

- Create questionnaire
- Add valid routing answer i.e. number, checkbox etc
- Create routing rule or skip condition
- Red box should surround selector with an error message below
